spec/models/ticket_spec.rb

```diff
@@ -192,7 +192,7 @@
     end
   end
 
-  describe 'currency updation' do
+  describe 'currency update' do
```

Comment:
how is this related to the change? it was also not in the cherry picked commit. :confused:

Reply:
It was from a second commit in the parent PR; I thought it trivial; but I can submit as a separate PR.